app/helpers/passengers_helper.rb

```diff
@@ -1,4 +1,4 @@
-module IndexHelper
+module PassenersHelper
```

Comment:
Yeahhhh I completely misunderstood what I was to be changing. I just pushed and it should be fixed now!